### ISSUE TITLE
[impl] function call

### ISSUE THREAD

**Issue by dannypsnl** [dannypsnl/elz#56]
Example:

```rust
fn add(l, r: i32) -> i32 {
  return l + r
}

fn main() {
  let a = add(1, 2)
}
```

**Comment by dannypsnl** [comment]
commit: e89473794c368a4d9bd2e26d6dda45ce6cab2ee8

**Comment by dannypsnl** [comment]
fix a bug at commit: 9e6c1252bbda4e9b5dd0e1617d4d51c2d880bcd1
